test/e2e/alertmanager_instance_namespaces_test.go

```diff
@@ -45,18 +45,28 @@ func testAlertmanagerInstanceNamespacesAllNs(t *testing.T) {
 	nonInstanceNs := framework.CreateNamespace(context.Background(), t, testCtx)
 	framework.SetupPrometheusRBACGlobal(context.Background(), t, testCtx, instanceNs)
 
-	_, err := framework.CreateOrUpdatePrometheusOperator(context.Background(), operatorNs, nil, nil, nil, []string{instanceNs}, false, true, true)
+	_, err := framework.CreateOrUpdatePrometheusOperator(
+		context.Background(),
+		operatorNs,
+		nil,
+		nil,
+		nil,
+		[]string{instanceNs},
+		false,
+		true, // clusterrole
+		true,
+	)
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	am := framework.MakeBasicAlertmanager(nonInstanceNs, "non-instance", 3)
+	am := framework.MakeBasicAlertmanager(nonInstanceNs, "non-instance", 2)
 	_, err = framework.MonClientV1.Alertmanagers(nonInstanceNs).Create(context.Background(), am, metav1.CreateOptions{})
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	am = framework.MakeBasicAlertmanager(instanceNs, "instance", 3)
+	am = framework.MakeBasicAlertmanager(instanceNs, "instance", 2)
```

Comment:
Any special reason to reduce the number of replicas?

I assume it is just because the test is valid for both 3 or 2 replicas and 2 replicas is just faster, but I wanted to ask anyway

Reply:
yes, the test doesn't care if it's 3, 2 or 1 replicas. But I'll revert it to avoid unrelated changes. We can have a look at all e2e tests globally and decide whether they really need multi replicas: in most cases, I think that the answer is no and switching to single replicas might save a few resources.